webform_import.module

```diff
@@ -373,14 +380,19 @@ function _webform_import_import($form, $form_state, $file) {
       continue 1;
     }
 
+    // Set defaults
     global $user;
     $uid = $user->uid;
     $ipaddress = webform_ip_address($node);
     $submitted = REQUEST_TIME;
+    $completed = REQUEST_TIME;
+    $modified = REQUEST_TIME;
     $sid = NULL;
+    $serial = NULL;  // Default NULL to identify rows with missing serial numbers
     $num = count($data);
 
     if ($arraylen == $num) {
+      // clear sub_array for new row / data record
```

Comment:
Inline comments should start with a capital letter and end with a period:

`// Clear the sub_array for new row or data record.`